### ISSUE TITLE
What is a denom variable in loss function?

### ISSUE THREAD

**Issue by adambelniak** [OctThe16th/PPO-Keras#11]
I don't see purpose of using  denom variable in loss function. Could you explain the reason of using it ?
